src/tests/zf_apps/zfsink.c

```diff
@@ -243,14 +245,75 @@ void print_attrs(struct zf_attr* attr)
 }
 
 
-static void monitor()
+static void zf_stats_header_print(struct zf_stack* stack,
+                                  zf_stats_collection* stats_collection)
+{
+  int i;
+
+  /* only display names for the first NIC and assume others are the same */
```

Comment:
this comment doesn't fit with the code immediately below it that is printing a new line for multiple NIC case

Reply:
There is a new line printed but the names output are only for the first NIC, hence `layout[0]` being used.

Reply:
Moving comment down to that code would be clearer.